src/status_im/contexts/onboarding/events.cljs

```diff
@@ -40,8 +40,9 @@
                [:screen/onboarding.enable-notifications :screen/onboarding.enable-biometrics]]}))
 
 (rf/reg-event-fx :onboarding/navigate-to-enable-notifications
- (fn [{:keys [db]}]
-   {:dispatch [:navigate-to-within-stack
+ (fn [{:keys [db]} from-screen]
+   {:db       (assoc db :onboarding/navigated-to-enter-seed-phrase-from-screen from-screen)
```

Comment:
could you pls elaborate on why `{:db (assoc db :onboarding/navigated-to-enter-seed-phrase-from-screen from-screen)` is needed here ?

Reply:
this function does not navigate from the new stack which is why I updated, the name of  db path the function could change to `onboarding/navigated-from-screen` alternatively I could also just `[:navigate-to-within-stack                                             [:screen/onboarding.enable-notifications :screen/onboarding.generating-keys]]` and this way we dont have to update the db